File: IDAUtil/Model/CO09Property.vb
Imports System.ComponentModel.DataAnnotations.Schema
<DebuggerDisplay("sku: {sku}, salesOrg: {salesOrg}, ATP: {ATP}")>
Public Class CO09Property
    <Column("[sku]")> Public Property sku As Long
    <Column("[salesOrg]")> Public Property salesOrg As String
    <Column("[plant]")> Public Property plant As Long
    <Column("[ATP]")> Public Property ATP As Long
    <Column("[recoveryDate]")> Public Property recoveryDate As String
    <Column("[recoveryQty]")> Public Property recoveryQty As String
End Class

File: IDAUtil/Model/Properties/TcodeProperty/CO09Obj/CO09Property.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Diagnostics;

namespace IDAUtil.Model.Properties.TcodeProperty.CO09Obj {
    [DebuggerDisplay("sku: {sku}, salesOrg: {salesOrg}, ATP: {ATP}")]
    public class CO09Property {
        [Column("[sku]")] public int sku { get; set; }
        [Column("[salesOrg]")] public string salesOrg { get; set; }
        [Column("[plant]")] public int plant { get; set; }
        [Column("[ATP]")] public double ATP { get; set; }
        [Column("[recoveryDate]")] public string recoveryDate { get; set; }
        [Column("[recoveryQty]")] public double recoveryQty { get; set; }
    }
}